applications/asset_tracker/overlay_carrier.conf

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Not sure if it's a well-established convention, but typically such overlay files use dash instead of underscore in the name:
`overlay-<type>`

Reply:
Right, that indeed is the convention. Changed.